src/components/common/Extensions/UserNavigation/MemberReputation/MemberReputation.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { FC } from 'react';
```

Comment:
Please add a partials directory and put there components that parts of the User Navigation (you can check the example in Extensions/Accordion component). Remember about changing displayName.

Reply:
Moved to partials, but kept them in folders to keep it clean